src/components/Nav.tsx

```diff
@@ -38,6 +57,22 @@ export default function Nav() {
       >
         Governance
       </Button>
+      <Menu
+        open={open}
+        onClose={handleClose}
+        anchorEl={governanceMenuEl}
+        TransitionComponent={Fade}
+        aria-controls={open ? "fade-menu" : undefined}
+        aria-haspopup="true"
+        aria-expanded={open ? "true" : undefined}
+      >
+        <MenuItem onClick={() => handleCloseAndNavigate("/proposals")}>
+          Proposals
+        </MenuItem>
+        <MenuItem onClick={() => handleCloseAndNavigate("/proposals/stake")}>
```

Comment:
Have we decided on the name "Staking"? If so, I would change the route to `/proposals/staking`.